### PR TITLE
Add an UI test case for capsule content counts granularity

### PR DESCRIPTION
**Pull request by vsedmik**
### Problem Statement
In 6.17 capsule content counts will be updated separately for each CV when it is synced to the capsule and we should have a test case to cover that.

There is currently a bug discovered by this test case, so the PRT is expected to fail on 2 assertions until the issue is resolved:
https://issues.redhat.com/browse/SAT-29679 

### Solution
This PR just adds UI coverage, more functional testing is added in CLI counterpart.

### PRT test Cases example
Requires
https://github.com/Katello/katello/pull/11176 (merged, in snap)
https://github.com/Katello/katello/pull/11187 (merged, in snap)
https://github.com/Katello/katello/pull/11235
https://github.com/SatelliteQE/airgun/pull/1605

```
trigger: test-robottelo
pytest: tests/foreman/ui/test_capsulecontent.py -k granular
airgun: 1605
Katello:
  katello: 11235
```

### CODE REVIEW

**Comment by LadislavVasina1** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/ui/test_capsulecontent.py -k 'test_positive_content_counts_granular_update'
airgun: 1605
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 9460
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_capsulecontent.py -k granular --external-logging
Test Result : ========== 1 passed, 2 deselected, 18 warnings in 2077.55s (0:34:37) ===========
```

File: tests/foreman/ui/test_capsulecontent.py [hunk 1]
```diff
@@ -234,6 +236,139 @@ def test_positive_content_counts_for_mixed_cv(
...
+        )
+        session.organization.select(org_name=function_org.name)
+
+        # 3. Ensure no counts were calculated yet for both CVs, both LCEs.
+        details = session.capsule.read_details(module_capsule_configured.hostname)
+        assert all(
+            details['content'][lce.name][cv.name]['expanded_repo_details'][1] == empty_counts
+            for lce in [lce1, lce2]
+            for cv in cvs
+        ), 'Expected "N/A" for content counts displayed'
```
Comment (pondrejk) [inline tests/foreman/ui/test_capsulecontent.py:324]: `not displayed`?

**Review by pondrejk — APPROVED** [review]
ack, minor question

File: tests/foreman/ui/test_capsulecontent.py (hunk 1 above)
Comment (vsedmik) [reply tests/foreman/ui/test_capsulecontent.py:324]: I meant something like `Expected "N/A" being displayed for the content counts`.
image

**Review by LadislavVasina1 — APPROVED** [review]
ACK

**Pull request merged**